bump/action.yml

```diff
@@ -35,23 +35,27 @@ inputs:
     required: false
   create-release:
     description: 'Create a GitHub Release if a version bump is performed (outside a pull request event)'
-    default: false
+    default: "false"
     required: false
   create-tag:
     description: 'Create a tag if a version bump is performed (outside a pull request event)'
-    default: false
+    default: "false"
     required: false
   version-prefix:
     description: 'Optional version prefix (eg. "", "v", "componentX-"). If set, only tags with this exact prefix shall be considered. "*" is a special value, meaning the closest version is used, regardless of prefix".'
     default: '*'
     required: false
   release-type:
     description: >-
-      The type of version increment to apply for this build when using the SdkVer version scheme. 
-      This has no effect when using the SemVer version scheme. 
+      The type of version increment to apply for this build when using the SdkVer version scheme.
+      This has no effect when using the SemVer version scheme.
```

Comment:
Same as before, please revert these and only leave the addition of the `create-changelog` mapping below (but remove the superfluous quotes around the boolean).